manifests/01_operator_configmap.yaml

```diff
@@ -6,4 +6,4 @@ metadata:
   annotations:
     release.openshift.io/create-only: "true"
 data:
-  disabled: "false"
+  disabled: "true"
```

Comment:
we don't want this disabled by default, do we?

Reply:
Uhhh no. :) Good catch.